Cabal-syntax/src/Distribution/Fields/ParseResult.hs

```diff
@@ -62,6 +63,17 @@ runParseResult pr = unPR pr emptyPRState failure success
     -- If there are any errors, don't return the result
     success (PRState warns (err : errs) v) _ = (warns, Left (v, err :| errs))
 
+-- | Chain parsing operations that involve 'IO' actions.
+liftPR :: (a -> IO (ParseResult b)) -> ParseResult a -> IO (ParseResult b)
+liftPR f pr = unPR pr emptyPRState failure success
```

Comment:
I'd prefer avoiding initialisms. `liftParseResult`?

Reply:
Yes, that's better :) https://github.com/haskell/cabal/pull/8889/files#diff-598f0bacc400f76cb4aba9b58fab4f5237fb21019eb0d6cd5f6c6dcfff3b068fR67